pennylane_qiskit/runtime_devices.py

```diff
@@ -55,6 +56,7 @@ class IBMQCircuitRunnerDevice(IBMQDevice):
     def __init__(self, wires, provider=None, backend="ibmq_qasm_simulator", shots=1024, **kwargs):
         self.kwargs = kwargs
         super().__init__(wires=wires, provider=provider, backend=backend, shots=shots, **kwargs)
+        self.runtime_service = QiskitRuntimeService(channel="ibm_quantum")
```

Comment:
What are the other possibilities for channel?

Reply:
per [the parameters in the docs](https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.QiskitRuntimeService.html), it's this or `ibm_cloud`. Everything I saw uses `ibm_quantum` so I am sticking with that